03/GeoffRiley/tags.py

```diff
@@ -0,0 +1,41 @@
+import re
+from collections import Counter
+from difflib import SequenceMatcher
+from itertools import permutations
+
+TOP_NUMBER = 10
+RSS_FEED = 'rss.xml'
+SIMILAR = 0.87
```

Comment:
again, not a good name. "SIMILAR" is a description and can be answered with yes or no and does not indicate a score or value. Better would be MIN_SIMILAR_VALUE or alike

Reply:
Again, this is in the boilerplate, perhaps a note that these elements are just suggestions rather than directions?  It took me some time examining the tests provided to work out what `SIMILAR` was supposed to represent anyway.